apps/src/storage/dataBrowser/DataBrowser.jsx

```diff
@@ -1,10 +1,200 @@
+import PropTypes from 'prop-types';
 import Radium from 'radium';
 import React from 'react';
+import color from '../../util/color';
+import msg from '@cdo/locale';
+import AddTableListRow from './AddTableListRow';
+import EditTableListRow from './EditTableListRow';
+import * as dataStyles from './dataStyles';
 
 class DataBrowser extends React.Component {
+  static propTypes = {
+    tableListMap: PropTypes.object.isRequired
+  };
+
+  state = {selectedTab: TabType.DATA_TABLES};
+
+  handleTabClick = newTab => this.setState({selectedTab: newTab});
+
+  emptyHandler = () => {};
```

Comment:
A comment might be useful here for why this empty handler is used?

Reply:
Just empty for now because I'm doing the UI pieces first and then I'll make everything actually work :D added a comment to make that more clear